### PR TITLE
IGVF-805-ui-biomarker-page

### PR DESCRIPTION
**Pull request by KAndreeva**
Added a few property to the biomarker page.

### CODE REVIEW

File: components/search/list-renderer/biomarker.js [hunk 1]
```diff
@@ -12,16 +12,18 @@ import {
 } from "./search-list-item";
 // lib
 import { getBiomarkerTitle } from "../../../lib/biomarker";
+import { getBiomarkerSearchIdentifier } from "../../../lib/biomarker";
 
 export default function Biomarker({ item: biomarker }) {
   const title = getBiomarkerTitle(biomarker);
+  const searchIdentifer = getBiomarkerSearchIdentifier(biomarker);
```
Comment (forresttanaka) [inline components/search/list-renderer/biomarker.js:19]: Change to `searchIdentifier`.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
🆒 Just a couple spelling corrections.

File: lib/biomarker.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 export function getBiomarkerTitle(biomarker) {
+  const titleElements = [
+    biomarker.classification,
+    biomarker.name,
+    biomarker.quantification,
+  ].filter(Boolean);
+  return titleElements.join(" ");
+}
+
+// Set up biomarker search page itentifier for top of item
```
[the remark this answers is not in the document]
Comment (KAndreeva) [reply lib/biomarker.js]: done

File: components/search/list-renderer/biomarker.js (hunk 1 above)
Comment (KAndreeva) [reply components/search/list-renderer/biomarker.js]: done

**Review by forresttanaka — APPROVED** [review]
👍

**Pull request merged**